docs/organizations/changing-your-plan-and-billing.md

```diff
@@ -15,19 +15,21 @@ Each organization on Codacy has a dedicated plan and associated billing. To make
 
 If you have any questions or need help with your account, please contact <mailto:support@codacy.com>.
 
-## Accepting new people to your organization
+## Allowing new people to join your organization
 
-**On Codacy Cloud**, organization owners control if team members need an approval before joining their organization. Codacy updates the billing automatically when new users join an organization.
+**On Codacy Cloud**, organization owners control if team members need an approval before joining their organization. Codacy updates your seats automatically when new users join an organization.
 
 !!! note
-    **If you're using GitHub Marketplace,** this configuration isn't available and team members must always wait for an organization owner to manually approve their requests to join the organization.
+    -   **If you're using GitHub Marketplace,** this configuration isn't available and team members must always wait for an organization owner to manually approve their requests to join the organization.
+
+    -   In some **Enterprise plans**, Codacy automatically adds to the organization new people that commit on your private repositories. However, they still need to join the organization on the Codacy app if they want to use the UI.
 
 Choose one of the following options in your organization **Settings**, page **Plan and billing**:
 
--   **Allow new people to join immediately:** team members with access to the organization on the Git provider can join the organization on Codacy immediately, as long as there are seats available.
+-   **Allow new people to join immediately:** team members with access to the organization on the Git provider can join the organization on Codacy app immediately, as long as there are seats available.
```

Comment:
I'm unsure if we should call it `Codacy app` or `the Codacy app`. If it's a product named `Codacy app`, we don't need the article. Thoughts?

Reply:
Right! It should be `the Codacy app`, as we have on the UI. I'll update it. 👍